src/components/DeliveryAddressComponent/DeliveryAddressComponent.jsx

```diff
@@ -39,25 +62,31 @@ const DeliveryAddressComponent = () => {
           </button>
         </div>
         <form className="delivery-main-box">
-          {addressList.map((element, index) => (
-            <DeliveryAddressBox
-              key={index}
-              index={index}
-              name={element.name}
-              mobileNumber={element.mobileNumber}
-              address={element.address}
-              city={element.city}
-              setAddressSelected={setAddressSelected}
-              addressSelected={addressSelected}
-              handleDelete={handleDelete}
-            />
-          ))}
+          {addressesList &&
+            addressesList.map((element, index) => (
```

Comment:
After discussing with Sergio about this, we do recommend this conditional rendering for mapping:
```js
{!!addressesList.map((element, index) => ( ...
```